spec/salesforce/donation_validator_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'salesforce/data_structs_for_tests'
+require 'salesforce/donation_validator'
+require 'spec_helper'
+
+module Salesforce
+  RSpec.describe DonationValidator do
+    let(:result) { validate_values(amount: '2000', account_id: '1') }
+    let(:fields) { result.item }
+
+    describe 'Salesforce required fields' do
+      it 'requires an amount' do
```

Comment:
These test describes are confusing. They aren't actually checking if a field is required, they are checking that the `fields` hash been populated

Reply:
They are documenting which fields are required by Salesforce or the application